Friesland data

Augustin4762 opened the pull request:
Data from Friesland municipalities added both for housing stock and housing price. Also, the housing price function added in the get_data function. Should solve the following issues: #47 and #45 .

Augustin4762 committed:
Friesland data

Data from Friesland municipalities added both for housing stock and housing price. Also, the housing price function added in the get_data function.

Augustin4762 committed:
updated test

Adjusted the lotr refference

ikbentimkramer commented on R/get_data.R:
```diff
@@ -20,6 +20,13 @@ get_data <- function (data_string) {
     saveRDS(res, data_path)
     return(res)
   }
+  if (data_string == "housing_price"){
+    res <- clean_housing_price_data(
+      read_housing_price_data(),
+      read_municipality())
+    saveRDS(res, data_path)
+    return(res)
+  }
```
Could you add a test for this part?

Augustin4762 replied:
I don't really get the mockery package, could you do it instead?

Augustin4762 replied:
I have tried again, can't get it done.

ikbentimkramer replied:
I will write some tests for this

ikbentimkramer replied:
> 
> 
> I will write some tests for this

Done

ikbentimkramer reviewed (changes requested):
There is no test for the part you added to the `get_data()` function. Could you add a test for this part please?

Augustin4762 committed:
Housing price plot

Added

Augustin4762 committed:
Revert "Housing price plot"

This reverts commit fbad47c54ede981acff8e84f01cb97b02e909bc4.

ikbentimkramer committed:
Add housing price tests for get data

ikbentimkramer committed:
Add cache folder to .Rbuildignore

ikbentimkramer committed:
Remove Rd creation for main_ui.R

ikbentimkramer committed:
Make get_data test check for specific error instead of an error

ikbentimkramer committed:
Fixes mockery error "cannot change value of locked binding"

I have no clue how or why this patch works, but changing the depth
of readRDS to 1 solved this error. There's a hint this works found
on https://github.com/r-lib/mockery/issues/30. A patch has been
created and has been merged into the repository, but has not found
its way into the CRAN release yet.
